### ISSUE TITLE
Warning on upgrade to v1.9.7

### ISSUE THREAD

**Issue by paulomcnally** [rmosolgo/graphql-ruby#2358]
The message:

```
Unable to define a helper for argument with name 'method' as this is a reserved name. If you're using an argument such as `argument method`, consider renaming this argument.
```

I'm not sure why this message appears.

**Comment by rmosolgo** [comment]
Hi, that's because one way to access input values is by methods, eg `input.method`. But, where an input argument conflicts with an existing method, we don't build a method for accessing the value. 

The problem is, there's no way to suppress that warning 🤦‍♂

**Comment by kspector** [comment]
@rmosolgo +1 for suppression of this warning - we're seeing a similar warning for aliasing a deprecated input argument to its successor using `as:`

**Comment by rmosolgo** [comment]
On master, you can use `method_access: false` to suppress this warning, please give it a try! It'll be part of the next 1.9.x release, too.

**Comment by kspector** [comment]
This works well - thanks!

**Comment by rusterholz** [comment]
This is supposedly fixed in #2369, but using v1.10.9 of this gem with v0.16.0 of [graphql-client](https://github.com/github/graphql-client), when calling `GraphQL::Client.load_schema` to load an external schema (JSON file) that has a field defined with a reserved name, we still get this warning (albeit with slightly altered text: "Unable to define a helper for argument with name 'values' as this is a reserved name. Add 'method_access: false' to stop this warning.")

That warning is unhelpful because it is not clear where we could pass `method_access: false` --the `load_schema` method [does not take any options](https://github.com/github/graphql-client/blob/v0.16.0/lib/graphql/client.rb#L47) nor pass any on to the methods it calls.

**Comment by rwarbelow** [comment]
@rusterholz We were stuck on this too. We ended up adding `method_access: false` to the class of the query:

```
module Queries
  class OurCustomQueryClass < Queries::BaseQuery
    argument :method, String, required: true, method_access: false
   ...
  end
end
```

I hope that helps!
